components/actions/TaskList.js

```diff
@@ -196,11 +228,18 @@ function TaskList(props) {
   function filterTasks(state, completed) {
     return sortedTasks
       .filter((item) => item.state === state)
-      .map((item) => (
-        <ListGroupItem key={item.id} className={`state--${item.state}`}>
-          <Task task={item} theme={theme} completed={completed} />
-        </ListGroupItem>
-      ));
+      .map((item) => {
+        const isLate = !completed && new Date(item.dueAt) < new Date();
```

Comment:
Extract `isLate` into a function (see above)?
Also, just a curious peculiarity: `new Date()` is always in the browser's local timezone, but `item.dueAt` is in the plan's local timezone. I don't think we need to do anything about it though, because it always works as intended from the customer's own browsers. Only from the other side of the world it might sometimes be one day off but no big deal.

Reply:
Renamed to isTaskLate and extracted into a reusable function.